blazor/maps/polygon.md

```diff
@@ -91,4 +91,174 @@ The following example shows how to customize the polygon shape over the geometry
     }
 }
 ```
-![Blazor Maps with polygon shape](./images/Polygon/blazor-map-polygon-shape-on-maps-layer.png)
\ No newline at end of file
+![Blazor Maps with polygon shape](./images/Polygon/blazor-map-polygon-shape-on-maps-layer.png)
+
+## Tooltip for polygon shape
+
+Tooltip is used to display more information about a polygon shape on mouse over or touch end event. This can be enabled by setting the [`Visible`]() property to **true** in the  [`MapsPolygonTooltipSettings`](). Additonally, you need to set the [`TooltipText`]() property in the [`MapsPolygon`](../api/maps/polygonSettingsModel) to show the tooltip text separately for each polygon shape, if you add 'n' numbers of polygon shapes.
```

Comment:
A tooltip is used to display more information about a polygon shape during a mouse or touch interaction. Tooltips and tooltip templates can be enabled by setting the [Visible]() property to **true** in the [MapsPolygonTooltipSettings](). Additionally, you need to set the desired content as a value to the [TooltipText]() property in the [MapsPolygon](../api/maps/polygonSettingsModel) tag to show the tooltip.

Reply:
Remove the code style in the properties.